src/rpc/blockchain.cpp

```diff
@@ -742,6 +745,23 @@ UniValue getblock(const UniValue& params, bool fHelp)
             "  \"nonce\" : n,           (numeric) The nonce\n"
             "  \"bits\" : \"1d00ffff\",   (string) The bits\n"
             "  \"difficulty\" : x.xxx,  (numeric) The difficulty\n"
+            "  \"chainSupply\": {          (object) information about the total supply\n"
+            "      \"monitored\": xx,           (boolean) true if the total supply is being monitored\n"
+            "      \"chainValue\": xxxxxx,      (numeric, optional) total chain supply\n"
+            "      \"chainValueZat\": xxxxxx,   (numeric, optional) total chain supply in satoshis\n"
```

Comment:
```suggestion
            "      \"chainValueZat\": xxxxxx,   (numeric, optional) total chain supply in zatoshis\n"
```
and three more times below.